search/includes/classes/class-search.php

```diff
@@ -901,6 +907,16 @@ public function filter__ep_formatted_args( $formatted_args, $args ) {
 		return $formatted_args;
 	}
 
+	public function truncate_search_string_length( $query_object ) {
+		if ( $query_object->is_search() ) {
+			$search = $query_object->query['s'];
```

Comment:
Minor - `WP_Query` also has a `get()`, so we could use that for consistency with the `set()` down below.